src/services/loop/readme.md

```diff
@@ -1,19 +1,46 @@
 Loop 
 ====
 
-Start the loopserver (Mock Swap Server) and loopclient (Rest API)
-```
+1. Start the loopserver (Mock Swap Server) and loopclient (Rest API)
+```sh
 make start-loop
 ```
 
-A successful loop server start returns this
+2. Configure the loop params in `default.yaml`
 
+```yaml
+loop:
+  maxOutboundLiquidityBalance: 2835916 # 0.03 ($600)
+  loopUrl: "https://localhost:8081"
+  loopoutAmount: 945305 # 0.009 ($200)
+  loopProvider: "@services/loop/providers/lightning-labs/loop-swap-provider" # or another custom provider, like Boltz
```

Comment:
I don't think when we have 2 swap providers that we will want to use one or the other. we should dynamically use both based on which has lowest fee or which is online. 

I know it's a hard problem to solve because we don't know in advance the routing fees.